nav2_map_server/include/nav2_map_server/map_saver.hpp

```diff
@@ -1,4 +1,4 @@
-// Copyright (c) 2018 Intel Corporation
+// Copyright (c) 2020 Samsung Research Russia
```

Comment:
Since map_saver was completely re-worked and all its previous functionality was moved to `map_io.hpp/.cpp`, I've moved Intel copyrights together with the code from newly updated map_saver.hpp/.cpp to map_io. Anyway, there is no problem to return copyrights back in order to respect all map_saver contributors.